docs/other/performance.md

```diff
@@ -0,0 +1,11 @@
+## Improving composer build performance
```

Comment:
I added this section because

1. I split the composer deployment setups into separate pages and linked both this page.
2. Thought we might extend it in the future.

Might be other performance related stuff in the docs that could be moved here also.